src/Docusaurus/docs/reference/heat/index.md

```diff
@@ -175,11 +175,24 @@ Example:
   </PropertyGroup>
 
   <ItemGroup>
-    <HarvestProject Include="..\MyProgram\MyProgram.csproj" />
+    <HarvestProject Include="..\MyProgram\MyProgram.csproj" ProjectOutputGroups="Binaries;Content;" />
+  </ItemGroup>
+
+  <!-- As soon as EnableProjectHarvesting set to true, Heat will try to Harvest all referenced projects. Notice the DoNotHarvest flag, this tells Heat not to do that. -->
+  <ItemGroup>
+    <ProjectReference Include="..\MyProgram\MyProgram.csproj" DoNotHarvest="true" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="WixToolset.Heat" />
   </ItemGroup>
 </Project>
 ```
+
```

Comment:
Since Heat generates these `RefIds`. It's good to mention the format it uses here.